docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -15,25 +15,25 @@ If you log in with GitHub, Codacy requires the following [app permissions](https
 
 Repository permissions:
 
--   Checks: Checks on code - Read & Write
--   Issues: Issues and related comments, assignees, labels, and milestones - Read & Write
--   Metadata: Search repositories, list collaborators, and access repository metadata - Read Only
--   Pull requests: Pull requests and related comments, assignees, labels, milestones, and merges - Read & Write
--   Webhooks: Manage the post-receive hooks for a repository - Read & Write
--   Commit statuses: Commit statuses - Read & Write
--   Administration: Create SSH keys - Read & Write. Codacy creates an SSH key on the repository to allow cloning and integrating with your repository.
+-   Checks - Read & Write. Codacy creates and updates check runs with the results of code analysis.
+-   Issues - Read & Write. Issues and related comments, assignees, labels, and milestones
```

Comment:
Don't know why we need this.

Reply:
![Screenshot 2021-04-13 at 09 30 09](https://user-images.githubusercontent.com/593860/114522026-da556d80-9c3a-11eb-8b92-a27ccf0ef474.png)
☝️